### ISSUE TITLE
Report: Account Details

### ISSUE THREAD

**Issue by kdjstudios** [kdjstudios/SABillingGitlab#495]
In GitLab by @kdjstudios on Oct 3, 2017, 14:30

**Submitted by:** Kyle

**Helpdesk:** NA

**Server:** All

**Client/Site:** All

**Account:** All

**Issue:** 

*  The report takes over ten seconds to load.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jan 4, 2018, 09:53

I just ran the CSV export of this report, and it appears that possibly there is an error in the export. It appears that the headers are not staying in the top row. 

Billings Site:
[account_details____1_.csv](/uploads/4cd0b08dc616ca6a66259e88316af87e/account_details____1_.csv)

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Jan 5, 2018, 07:45

@kdjstudios 

I might be wrong but, what I understood is that, report headers are shifting down..?? But, in the attached csv the top row is for the site name and the second row serves as a header of the report.

Report taking over 10s to load is a bigger issue, can you please tell us which report you have pulled..?

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jan 5, 2018, 09:54

@smasih @tim.traylor 

The first row consists of the site.

In the report the 2nd row contains about 1/4 of the headers. In the 3rd row is the value to those headers. 

However, in the 4th row there are 1/4 more of the headers, which I would expect should be a continuation of the 2nd row headers. There are no values for these headers. 

There is then in the 5th row with the about 1/4 of the headers. There are values in the 6th row it appears for those headers. 

Lastly, in the 7th and 8th rows there are last 1/4 of the headers with no values under these. 

This then repeats for every single account.

In summary it appears that each account should only have one row of headers and one row of values for those headers. There also appears to be duplicate headers listed multiple times. 

I have highlighted the row in green that should contain all of the headers, below that I have highlighted in yellow the row that should contain the values. I then highlighted all additional rows in red that should be combined into the above two rows. The additional row highlighted in yellow it to help show that out of the four additional rows of headers that only one of them has values. 

![Untitled](/uploads/03c0779aca60031fa7dd96d0c07eff0f/Untitled.png)

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jan 5, 2018, 09:55

This is marked for discussion to set a priority. Please do not work on this, until then.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Jan 6, 2018, 24:10

@kdjstudios 

Thanks for this excellent explanation, now I understand what you were trying to say earlier. Although I am  not sure which report  is this.

**Comment by kdjstudios** [comment]
In GitLab by @smasih on Mar 7, 2018, 23:52

@kdjstudios 

please assign priority to this, we can pipeline it for the coming release.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Mar 9, 2018, 08:49

Marking for report project. I think this will be part of that.
